apps/asap-server/src/controllers/teams.ts

```diff
@@ -217,14 +208,10 @@ export default class Teams implements TeamController {
 
     const { total, items: teams } = queryTeamsContentsWithTotal;
 
-    const teamUsers = await Promise.all(
-      teams.map((team) => fetchUsers(team.id, this.users.client)),
-    );
-
-    const teamItems = teams.map((team, index) =>
+    const teamItems = teams.map((team) =>
       transformGraphQLTeam(
         team,
-        transformRestTeamMember(teamUsers[index], team.id),
+        transformGraphQLUser(team?.referencingUsersContents || [], team),
```

Comment:
actually - since you're changing the logic so much - I wonder if we need to pass down the members separately when they are still part of the `team`? like.. whats the point of extracting the members as a separate argument when the `GraphqlTeam` type has a reference to the team members? As far as I can tell `transformGraphQLTeam` function could get those members from the first argument